server/conf/env-var-docs.json

```diff
@@ -707,6 +707,11 @@
         "mode": "HIDDEN",
         "description": "Enables caching for questions and their associated data.",
         "type": "bool"
+      },
+      "ENHANCED_OIDC_LOGOUT_ENABLED": {
+        "mode": "ADMIN_WRITEABLE",
```

Comment:
**suggestion:** We may want to wait to make this writeable until the feature is ready to be used. In the meantime, we could use "ADMIN_READABLE" or hide it with "HIDDEN".

Reply:
Good idea, changed to "HIDDEN".